Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [package]
 name = "fslock"
-version = "0.2.1"
-authors = ["brunoczim <brunoczim@gmail.com>"]
-edition = "2018"
+version = "0.2.2"
+authors = ["brunoczim <brunoczim@gmail.com>", "Fredrick Brennan <copypaste@kittens.ph>"]
+edition = "2021"
```

Comment:
I think edition bumping count as breaking change, as it also bumps minimum required rustc version